radicle-cli/tests/commands.rs

```diff
@@ -285,42 +285,6 @@ fn rad_rm() {
     test("examples/rad-rm.md", working.path(), Some(home), []).unwrap();
 }
 
-#[test]
-fn rad_track() {
-    let mut environment = Environment::new();
-    let alice = environment.node("alice");
-    let working = tempfile::tempdir().unwrap();
-    let alice = alice.spawn(Config::default());
-
-    test(
-        "examples/rad-track.md",
-        working.path(),
-        Some(&alice.home),
-        [],
-    )
-    .unwrap();
-}
-
-#[test]
-fn rad_clone() {
-    logger::init(log::Level::Debug);
-
-    let mut environment = Environment::new();
-    let mut alice = environment.node("alice");
-    let bob = environment.node("bob");
-    let working = environment.tmp().join("working");
-
-    // Setup a test project.
-    let _ = alice.project("heartwood", "Radicle Heartwood Protocol & Stack");
-
-    let alice = alice.spawn(Config::default());
-    let mut bob = bob.spawn(Config::default());
-
-    bob.connect(&alice).converge([&alice]);
-
-    test("examples/rad-clone.md", working, Some(&bob.home), []).unwrap();
-}
```

Comment:
Can we keep these tests?

Reply:
Thanks for catching this.  I left them out by mistake from my earlier correction to include the old tests.